libs/src/services/contracts/ProviderSelection.js

```diff
@@ -0,0 +1,79 @@
+
+const Web3 = require('web3')
+
+const ServiceSelection = require('../../service-selection/ServiceSelection')
+const EthWeb3Manager = require('../../../src/services/ethWeb3Manager/index')
+
+/**
+ * This class provides the logic in selecting the proper gateway
+ */
+// TODO: use env vars idk how to use env vars lol
```

Comment:
You'll need to set this in the dapp in
.env.prod and .env.staging

Inside AudiusBackend.js in the dapp, you can access them with process.env.REACT_APP_VARIABLE_NAME

You'll then want to pass those into the web3 config in the libs ctor/init.